src/main/java/com/afornalik/ox/board/ChangeFieldStatusException.java

```diff
@@ -0,0 +1,4 @@
+package com.afornalik.ox.board;
+
+public class ChangeFieldStatusException extends Exception {
+}
```

Comment:
What is the purpose of this class? And why is it empty?

Reply:
I delete it here but i use it in next feature.